packages/components/src/components/data-table/_data-table-action.scss

```diff
@@ -155,6 +155,7 @@
     .#{$prefix}--search
     .#{$prefix}--search-input:focus {
     @include focus-outline('outline');
```

Comment:
Would this mean that we don't need the focus-outline mixin anymore, or no? 👀

Reply:
we probably still want to keep it for high contrast mode in non IE browsers

Reply:
Should we add a new entry to the `focus-outline` mixin to handle these cases?

```scss
@if ($type == 'boxshadow') {
    box-shadow: inset 0 0 0 2px $focus;
  }
```

Reply:
I'm not sure about if that should go in the mixin since it seems we use box shadow for outlines differently in a few places, and I think the primary reason is also for IE compatibility

Reply:
Makes sense, thanks for clarifying 👍